packages/interpretations/src/components/Comment/__tests__/Comment.spec.js

```diff
@@ -112,20 +108,4 @@ describe('components: Comment -> Comment component ', () => {
         });
     });
 
-    describe('with prop dialogIsOpen as false', () => {
```

Comment:
Can these deleted tests be replaced with something else or are they completely obsolete?

Reply:
Or is that what's now being tested in `CommentsList.spec.js`?

Reply:
It's being tested in `CommentsList.spec.js`.